gradle.properties

```diff
@@ -13,7 +13,7 @@ commons_collections_version = 3.2.1
 commons_io_version = 1.4
 commons_lang_version = 3.3.2
 commons_text_version = 1.1.8
-cq_workflow_console_version = 5.8.2
+cq_workflow_console_version = 5.6.6
```

Comment:
Should probably add a comment acknowledging that it's an "old" version, but needed to compile because of the change in obfuscation policies.